### ISSUE TITLE
gluon.data.ArrayDataset TypeError

### ISSUE THREAD

**Issue by DeepInDeeper** [ypwhs/DogBreed_gluon#2]
使用你的代码运行时，
> data_iter_224 = gluon.data.DataLoader(gluon.data.ArrayDataset(X_224), batch_size=batch_size)       
 
查看资料，关于gluon.data.ArrayDataset需要两个参数 ?请问是什么问题

> mxnet.gluon.data.ArrayDataset(data, label) [link](https://mxnet.incubator.apache.org/api/python/gluon.html#mxnet.gluon.data.ArrayDataset)

**Comment by ypwhs** [comment]
去掉即可
